component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -283,6 +286,35 @@ private boolean validateJWTInDataBase(String jti, long currentTimeInMillis,
         return true;
     }
 
+    /**
+     * This method is to get JTI from the DB.
+     * For the migration purposes (preserve existing behaviour),
+     * We are searching for the current tenant and default tenant.
+     *
+     * @param jti      JTI.
+     * @param tenantId Tenant id.
+     * @return JWT entry if exists.
+     * @throws OAuthClientAuthnException OAuthClientAuthnException.
+     */
+    private JWTEntry getJTIfromDB(String jti, int tenantId) throws OAuthClientAuthnException {
+
+        List<JWTEntry> jwtEntries = jwtStorageManager.getJwtsFromDB(jti, tenantId);
+        if (jwtEntries.isEmpty()) {
+            return null;
```

Comment:
Don't we need to return an error if the 'jwtentries' is null?

Reply:
If its null then there is no entry in DB.
Need this null here......
https://github.com/wso2-extensions/identity-oauth-addons/blob/master/component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java#L269